protocol.md

```diff
@@ -351,6 +351,20 @@ If the length of the upload exceeds the maximum, which MAY be specified using
 the `Tus-Max-Size` header, the Server MUST respond with the
 `413 Request Entity Too Large` status.
 
+The Client MAY include the entire or a chunk of the data, which is meant to be
+uploaded, in the body of the `POST` request. In this case, similar rules as for
+the `PATCH` request and response apply. The Client MUST include the
+`Content-Type: application/offset+octet-stream` header. The Server SHOULD accept
+as many bytes as possible and MUST include the `Upload-Offset` in the response
+and MUST set its value to the offset of the upload after applying the accepted
+bytes. If the Server supports the functionality described in this paragraph, it
+MUST advertise this by including the `creation-with-chunk` value to the
+`Tus-Extension` header. The Client SHOULD verify that the Server is capable of
+handling this cases before sending the `POST` request. In addition, the Client
```

Comment:
What if we said:
The Client SHOULD verify that the Server is capable of handling file data in the Create body before sending the `POST` request.

If not, I believe it should be either "this case" or "these cases"